definitions/features/installer.rb

```diff
@@ -105,13 +105,20 @@ def installer_arguments
     installer_args = ''
 
     if feature(:foreman_proxy) &&
-       feature(:foreman_proxy).with_content?
+       feature(:foreman_proxy).with_content? &&
+       !feature(:katello_install) &&
```

Comment:
This looks odd. Why not `katello_install`? Katello itself also has `--disable-system-checks`.